Userland/Libraries/LibGfx/SystemTheme.h

```diff
@@ -132,6 +136,17 @@ inline const char* to_string(ColorRole role)
     }
 }
 
+enum class FlagRole {
+    NoRole,
+
+#undef __ENUMERATE_FLAG_ROLE
+#define __ENUMERATE_FLAG_ROLE(role) role,
+    ENUMERATE_FLAG_ROLES(__ENUMERATE_FLAG_ROLE)
+#undef __ENUMERATE_FLAG_ROLE
+
+        __Count,
```

Comment:
This indentation looks really weird - sudden jump from 4 to 8 spaces?

Reply:
Clang-format does that for all the FooRole enums for some reason. :yakshrug:  ~~KlingFormat wen~~